### DISCUSSION TITLE
Welcome to capacitor-android-foreground-service Discussions!

### DISCUSSION THREAD

**Discussion by robingenz** [capawesome-team/capacitor-android-foreground-service#4] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ababbef** [comment]
hi, 
I wanted to know that the background service plugin working effectively even when the application is closed. It's impressive how it continues to run even when the application is closed. I just have a question that can we reactivate it every two minutes to retrieve data from the BLE?

**Reply by robingenz** [reply]
Hi @ababbef,

> I just have a question that can we reactivate it every two minutes to retrieve data from the BLE?

What do you mean exactly? I don't fully understand your use case.

**Comment by ababbef** [comment]
Hi,
I hope this message finds you well. I am reaching out to request a background service that can efficiently read data from a Bluetooth Low Energy (BLE) device at regular intervals.

Specifically, I would like the service to retrieve data from the BLE device every 2 minutes. This data is crucial for my project, and having a reliable and automated background service would greatly enhance the efficiency of data collection.

Sent from Mail for Windows

**Reply by robingenz** [reply]
Hi @ababbef,
unfortunately I do not offer any custom implementation for this plugin.
Your use case is very special.
This plugin is only used to create an Android Foreground service.

**Comment by ababbef** [comment]
I hope this message reaches you in good health. I am writing to follow up on my previous request to remove my sponsorship from “ababbef”. As of now, my removal request is still pending, and I kindly ask for your prompt assistance in processing this request.

 

To reiterate, I no longer wish to continue sponsoring Capawesome on GitHub. I understand that it may take some time to complete the necessary administrative steps, but I would greatly appreciate it if you could expedite the process. Please confirm the successful removal of my sponsorship at the earliest convenience.

 

If there are any additional steps or information required from my end to facilitate this removal, please let me know, and I will gladly provide the necessary details.

 

Thank you for your understanding and prompt attention to this matter. I look forward to receiving confirmation of the sponsorship removal from GitHub.

 

Sincerely,

Abdullah

**Reply by robingenz** [reply]
Thank you for your request. You have to cancel your GitHub sponsorship on your own via your GitHub account. You can find more information here: [Canceling a sponsorship](https://docs.github.com/en/billing/managing-billing-for-github-sponsors/downgrading-a-sponsorship#canceling-a-sponsorship)

**Comment by ababbef** [comment]
Thanks your sending guidence, I already done with that. But still showing that I im as sponsoring so that’s why I was sending you email. Here are some screen clips. 



can you please confirm that, I’m successfully unsponsored with you guys. 

Thanks

**Reply by robingenz** [reply]
I cannot see any screen clips. Please send us an email to [support@capawesome.io](mailto:support@capawesome.io).
